### PR TITLE
Document NDK_HOME, build directory

### PR DESCRIPTION
**Pull request by solidsnack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ Before you can compile for Android, you need to setup your environment. This nee
  - Update the SDK: `./android-sdk-linux/tools/android update sdk -u`
 
  - Install `cargo-apk` with `cargo install cargo-apk`.
+ - Be sure to set `NDK_HOME` to the path of your NDK.
 
 ## Compiling
 
-Run `cargo apk`.
+Run `cd glue && cargo apk`.
```
Comment (tomaka) [inline README.md]: The `Compiling` section here means "compiling your project", not "compiling this library".
I guess you could put something like "Go to your project's directory and run `cargo apk`" if it's unclear.
Compiling the glue with `cargo apk` doesn't really make sense.

File: README.md [hunk 2]
```diff
@@ -14,10 +14,11 @@ Before you can compile for Android, you need to setup your environment. This nee
  - Update the SDK: `./android-sdk-linux/tools/android update sdk -u`
 
  - Install `cargo-apk` with `cargo install cargo-apk`.
+ - Be sure to set `NDK_HOME` to the path of your NDK.
```
Comment (tomaka) [inline README.md]: Please add the fact that `ANDROID_HOME` must point to the path of the SDK as well. There's no reason to say something for the NDK but not for the SDK.

File: README.md (hunk 1 above)
Comment (solidsnack) [inline README.md]: Sorry; I have no idea what I am doing. Is there an "Android-Rust example project" repo I should be looking at? I was following these steps to validate my toolchain before starting a project.

Comment (tomaka) [inline README.md]: > Is there an "Android-Rust example project" repo I should be looking at?

The advantage of this library is that there's no need to do something special in your project. Just go to the path of any Cargo.toml and run `cargo apk`.

If you want graphics (which you probably do), then you can take a look at a library that provides graphics: piston, glium, gfx, etc.

Comment (solidsnack) [inline README.md]: > The advantage of this library is that there's no need to do something special in your project.

In truth I am really out of my depth, then. I assumed an APK must be "an app" with a UI, &c, and therefore must use native libs. Can this library be used to provide a Rust lib _to_ a Java/Android application?

Comment (tomaka) [inline README.md]: > Can this library be used to provide a Rust lib to a Java/Android application?

No. To do so, you would compile your library as a regular library, with `cargo build --target arm-unknown-androideabi`.

**Comment by tomaka** [comment]
Thanks!

**Pull request merged**